registration.php

```diff
@@ -14,518 +35,621 @@
+                                            ($update) ? $row['employer'] : (isset($_POST['employer']) ? 
+                                            $inputData['postData']['employer'] : ''); ?> ">
+                                        <span class="error"><?php echo $errorList['employer'];?>
+                                        </span>
+                                    </div>
+                                </div>
+                                <!-- Image Upload -->
+                                <div class="form-group">
+                                    <label class="col-lg-2 col-md-2 col-sm-2 col-xs-12" 
+                                        for="textinput">Upload Image
+                                    </label> 
+                                    <input type="file" name="image"  value="<?php echo ($update) ? 
+                                        $row['image'] : (isset($_POST['image']) ? $_POST['image'] : 
+                                        ''); ?>" />
+                                    <?php
+                                        if ($update)
+                                        {
+                                    ?>
+                                            <!-- Modal -->
+                                            <a href="" data-target="#empImage" data-toggle="modal">
+                                                Current Picture</a>
+                                            <div id="empImage" class="modal fade" role="dialog">
+                                                <div class="modal-dialog">
+                                                    <!-- Modal content-->
+                                                    <div class="modal-content">
+                                                        <div class="modal-header">
+                                                           <h4 class="modal-title">Your Image</h4>
+                                                        </div>
+                                                        <div class = "modal-body">
+                                                            <img src = "<?php 
+                                                                echo IMAGEPATH.($update ? 
+                                                                $row['image'] : (isset
+                                                                ($_POST['image']) ? $_POST['image'] : 
+                                                                '')); ?>" alt = "No image" 
+                                                                height = "300" width = "500">
+                                                        </div>
+                                                        <div class="modal-footer">
+                                                            <button type="button" class="btn 
+                                               
```

Comment:
This if else block can be reduced like
<?php
$button_submit = ($update) ? 'Update' : 'Submit';
$button_reset = ($update) ? 'Clear' : 'Reset';
?>
<button type="submit" class="btn btn-success" role="button"><?php echo $button_submit; ?></button>
<button type="reset" class="btn btn-primary" role="button"><?php echo $button_reset; ?></button>